frontend/viewer/src/lib/about/AboutDialog.svelte

```diff
@@ -26,7 +26,7 @@
   </Button>
   <Dialog {open} on:close={toggleOff} class="w-[700px]">
     <div class="m-6 prose">
-      <Markdown md={$text} plugins={[{ renderer: { a: NewTabLinkRenderer } }]} />
+      <Markdown md={$text} plugins={[{ renderer: { a: NewTabLinkRenderer as unknown as Component } }]} />
```

Comment:
this should use the new markdown component

Reply:
The new Markdown component is in the lexbox frontend and I haven't implemented it in the viewer. I'll implement a copy of it in the viewer, because we might indeed end up wanting to use it more often.

Reply:
Done in commit fd453fbf.